src/pages/__tests__/test.date.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import DateMicro from 'components/MicroComp/datemicro'
+import { shallow } from 'enzyme'
+import schema2 from 'components/Schema/basicschema2'
+import moment from 'moment'
+
+const meta = { id: 5, label: 'DateCreated', renderer: 'date', key: ['dateCreated'] }
+test('DateMicro Renders Properly', () => {
+  const wrapper = shallow(<DateMicro schema={schema2} meta={meta} />)
+  expect(wrapper.find('dt').props().className).toEqual(meta.key)
+  expect(wrapper.find('dd').text()).toEqual(moment(schema2[meta.key]).format('YYYY-MM-DD'))
```

Comment:
Let's not import `schema2` and `moment` here but instead hard code in some test schema values into the test itself and test the `toEqual` to the literal expected output.